text/XXXX-delete_and_drop.md

```diff
@@ -0,0 +1,225 @@
+
+Users should come to think of `--amend` as a shortcut:
+
+```
+# two commands:
+$ qri delete me/dataset --revisions 1
+$ qri save me/dataset
+
+# same thing, one command:
+$ qri save me/dataset --amend 
+```
+
+When a user understands these two things to be equal, it'll help solidfy their mental model of versioning with Qri.
+
+
+### the drop command
+Drop removes other people's datasets. By default drop operates on your local node, removing a dataset that you've pulled. Drop can be paired with the `--remote` flag to ask remotes to drop a dataset.
+
+The `drop` flag answers two common scenarios:
+* "I was using this dataset and don't need it anymore": `$ qri drop user/dataset`
+* "I made a dataset, and want it off qri.cloud": `$ qri drop me/dataset --remote registry`
+
+Drop joins `push` and `pull` as a family of active-verb commands that describe a "me and them" interaction pattern.
+
+```
+$ qri drop --help
+Drop removes datasets you've pulled. After running drop, a dataset will no
+longer show up in your local list, and all stored versions of that dataset will
+be deleted.
+
+Use the --remote flag to asks a remote to completely remove a dataset, 
+deleting history and stored versions. Drop with the remote flag reverses a push.
+
+Drop only works on datasets you *don't* own. Use the delete command to remove
+datasets you've created.
+
+Drop completely removes a dataset, which may not be what you'd like do do. 
+If you want to...
+* delete a dataset you own:
+  $ qri delete me/dataset --all
+* remove a version from a remote, for a dataset you own:
+  $ qri delete me/dataset --revisions 1
+  $ qri push [remote] me/dataset
+* drop a specific version from a remote:
+  $ qri storage drop me/dataset@[version] --remote [remote]
+* unlist a dataset on a remote:
+  $ qri access unpublish me/dataset
+  $ qri push [remote] me/dataset
+
+Usage:
+  qri drop DATASET [DATASET...] [flags]
+
+Examples:
+  # Drop a dataset you've pulled
+  $ qri drop user/dataset
+
+  # Drop a few datasets at once
+  $ qri drop user/dataset other_user/other_dataset other_user/another_dataset
+
+  # Remove a dataset from the registry
+  $ qri drop me/annual_pop --remote registry
+
+  # Remove a dataset from a remote you've pushed to named "sally"
+  $ qri drop me/annual_pop --remote sally
+
+Flags:
+  -h, --help               help for drop
+      --remote             send a drop request to a remote
+```
+
+`drop` and `delete` have different audiences. `drop` is for data _consumers_, removing datasets they've pulled. `delete` is for data _authors_, correcting data they're pushing. Attempting to drop a dataset you own will error:
+
+```
+$ qri drop me/dataset
+error: cannot drop a dataset you own
+
+because you are the author of this dataset, dropping it would remove the 
+authoritative history of this dataset. Instead of dropping, you can 
+delete this dataset:
+
+  $ qri delete --all me/dataset
```

Comment:
Is there meaningful reason to worry that this isn’t quite the same thing? (At least if I’m understanding right.) Isn’t there a not-unusual (but probably not super common) scenario where I push to a registry and then drop the dataset from my machine? If I use `qri delete` for that instead, aren’t I left with a local dataset whose history diverges from the remote one? (Since it’s now a dataset in my repo but with 0 commits.)